packages/dx-react-scheduler-demos/src/demo-sources/scheduler-date-navigator/material-ui/date-navigator-controlled.jsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import Paper from '@material-ui/core/Paper';
+
+import { ViewState } from '@devexpress/dx-react-scheduler';
+
+import {
+  Scheduler,
+  WeekView,
+  Toolbar,
+  DateNavigator,
+  Appointments,
+} from '@devexpress/dx-react-scheduler-material-ui';
+
+import { appointments } from '../../../demo-data/appointments';
+
+export default class Demo extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data: appointments,
+      currentDate: '2018-06-27',
+    };
+    this.currentDateChange = (currentDate) => { this.setState({ currentDate }); };
+  }
+
+  render() {
+    const { data, currentDate } = this.state;
+
+    return (
+      <Paper>
+        <Scheduler
+          data={data}
+        >
+          <ViewState
```

Comment:
I don't like an idea to extract plugin with core functionality. Seems like the content of the ViewState plugin should be merged with Scheduler. I don't think that we should merge these things in that PR, but we should consider this change for the final release.